docs/configuration.md

```diff
@@ -2288,6 +2288,13 @@ showDF(properties, numRows = 200, truncate = FALSE)
     on the receivers.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.concurrentJobs</code></td>
+  <td>1</td>
+  <td>
+    The number of concurrent jobs.This parameter directly affects the number of threads in the jobExecutor thread pool.
```

Comment:
I think we don't document this on purpose, as its behavior is hard to understand. This description doesn't really help a caller, anyway.

Reply:
@srowen I see, but this more or less help us uderstand this configuration. How can we stop eating for fear of choking? To better explain its behavior,maybe we could make this description more specific rather than giving it up.